src/GRA.Web/Areas/MissionControl/Views/Programs/Detail.cshtml

```diff
@@ -158,7 +160,14 @@
                     </div>
                 </div>
             </div>
-
+            <div class="row row-spacing">
+                <div class="col-xs-12">
+                    <label asp-for="BadgeAltText" class="control-label"></label>
+                    <span class="fa fa-info-circle wide-tooltip" title="@SharedLocalizer[GRA.Annotations.Info.AltTextExplanation, SharedLocalizer[GRA.Annotations.Interface.Badge]]" data-toggle="tooltip" data-placement="top"></span>
+                    <textarea asp-for="BadgeAltText" class="form-control">@Model.BadgeAltText</textarea>
+                    <span asp-validation-for="BadgeAltText" class="text-danger"></span>
+                </div>
+            </div>
```

Comment:
Handling of this field doesn't seem to be added in the controller.

Reply:
Added in commit f6c82f256a68ad5ebada2c3db53c3f5739b8f387